Header header
string name
string[] things
